jac-uk/admin
# #2012 Exercise Approval - Help Text - UT Feedback

## What's included?
Updated approval modal text to condense the message so not so repetitive.
Single line followed by list of teams.

Closes #2012 

## Who should test?
✅ Product owner
✅ Developers
✅ UTG

## How to test?
When submitting an exercise for approval ensure the text in the modal (popup) says:

'Please make the information page and application form (if relevant) has been cleared by the following bodies before submitting for approval.'

Followed by a list of the required bodies.

## Risk - how likely is this to impact other areas?
🟢 No risk - this is a self-contained piece of work

---
PREVIEW:DEVELOP
_can be OFF, DEVELOP or STAGING_
